src/etl/ReportingETL.cpp

```diff
@@ -912,6 +912,171 @@ ReportingETL::monitor()
         }
     }
 }
+bool
+ReportingETL::loadCacheFromClioPeer(
+    uint32_t ledgerIndex,
+    std::string const& ip,
+    std::string const& port,
+    boost::asio::yield_context& yield)
+{
+    BOOST_LOG_TRIVIAL(info)
+        << "Loading cache from peer. ip = " << ip << " . port = " << port;
+    namespace beast = boost::beast;          // from <boost/beast.hpp>
+    namespace http = beast::http;            // from <boost/beast/http.hpp>
+    namespace websocket = beast::websocket;  // from
+    namespace net = boost::asio;             // from
+    using tcp = boost::asio::ip::tcp;        // from
+    try
+    {
+        boost::beast::error_code ec;
+        // These objects perform our I/O
+        tcp::resolver resolver{ioContext_};
+
+        BOOST_LOG_TRIVIAL(trace) << __func__ << "Creating websocket";
+        auto ws =
+            std::make_unique<websocket::stream<beast::tcp_stream>>(ioContext_);
+
+        // Look up the domain name
+        auto const results = resolver.async_resolve(ip, port, yield[ec]);
+        if (ec)
+            return {};
+
+        BOOST_LOG_TRIVIAL(trace) << __func__ << "Connecting websocket";
```

Comment:
this and many other lines seem to collapse `__func__` and the string after it (needs a space)